<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add word</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qword.qrc">
    <normaloff>:/images/qword.svg</normaloff>:/images/qword.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Foreign tongue:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtEs"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mother tongue: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRu"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cboxRevert">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Reverse translation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="qword.qrc">
         <normaloff>:/images/1238844468_next.png</normaloff>:/images/1238844468_next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="qword.qrc">
         <normaloff>:/images/1238844450_fileclose.png</normaloff>:/images/1238844450_fileclose.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qword.qrc"/>
 </resources>
 <connections/>
</ui>
